Replicator/c4ReplicatorImpl.hh

```diff
@@ -485,9 +485,11 @@ namespace litecore {
                 }
             }
 
-            auto onStatusChanged = _onStatusChanged.load();
-            if (onStatusChanged && status.level != kC4Stopping /* Don't notify about internal state */)
-                onStatusChanged(this, status, _options->callbackContext);
+            if( !(status.error.code && status.level > kC4Offline) ) {
+                auto onStatusChanged = _onStatusChanged.load();
+                if (onStatusChanged && status.level != kC4Stopping /* Don't notify about internal state */)
+                    onStatusChanged(this, status, _options->callbackContext);
+            }
```

Comment:
status.level != kC4Stopping -> means that it will only report when the status == kC4Offline after this change.

Why not just do `if (onStatusChanged && status.level == kC4Offline` and remove the outer if?

Reply:
No, it will report kC4Stopped and kC4Offline. (@pasin, I think you mixed up kC4Stopped and kC4Stopping.)

Reply:
Right, that is kC4Stopping not kC4Stopped.